src/core/forces.cpp

```diff
@@ -154,7 +154,7 @@ void force_calc() {
 
 #if defined(LB_GPU) || defined(LB)
 #ifdef LB
-  if (lattice_switch & LATTICE_LB) {
+  if (lattice_switch == ActiveLB::CPU) {
```

Comment:
as in #2566 id prefer to move the enum to the LB interface and only expose via an interfacefunction